### PR TITLE
Fix sharing buttons

### LINKED ISSUE #274: [frontend][web] Optimize share buttons ruler
Reorder sharing buttons to:
1- Facebook
2- Whatsapp
3- Facebook messenger
4- Linkedin
5- Twitter
6- Copy link+text/copy

Add the ones that do not exist yet.

### PR DESCRIPTION
**Pull request by ruijadom**
- i created the 3 sharing buttons elements missing.

1. whatsapp
2. facebook-messenger
3. copy-link-text

- media queries for less margin-right in mobile
- updated de video model
- 2 functions for trigger the sharing and 1 for the copy

note:
when I ran the code for the first time I noticed an error in the console:
"FB is not defined." 
because of the previous implementation of sharing on facebook.

For that i have install "@types/facebook-js-sdk": "3.3.0" and update the ts.config.app.json     
"types": ["facebook-js-sdk"]

### CODE REVIEW

**Comment by diogoteix** [comment]
In Iphone 5 I got this
image

**Comment by ruijadom** [comment]
> In Iphone 5 I got this
> image

**Comment by ruijadom** [comment]
thanks @diogoteix ! now is fixed

**Comment by diogoteix** [comment]
When we are sharing Whatsapp, messenger or copy/paste we are only sharing the text
Shouldn't we also share the video link or one from the API like the other types of share?

File: Trackovid19-web/src/app/shared/share-status/share-status.component.html [hunk 1]
```diff
@@ -46,9 +47,23 @@
...
-            <img src="assets/img/icons/twitter-share-icon.svg" />
+
+          <!-- Whatsapp -->
+          <div *ngIf="video.share" id="whatsapp" class="icon-share" (click)="shareWhatsapp()">
+            <img src="assets/img/icons/whatsapp-share-icon.svg" />
           </div>
+
+          <!-- Facebook messenger -->
+          <div
+            *ngIf="video.share"
```
Comment (fabioscsilva) [inline Trackovid19-web/src/app/shared/share-status/share-status.component.html]: Can you update all the new 3 icons to check if is to show like the others?
For instance this one: 
```
*ngIf="video.share.messenger"
```
we will update the API now to use this codes:

- messenger
- whatsapp
- copyText

**Review by fabioscsilva — APPROVED** [review]

File: Trackovid19-web/src/app/shared/share-status/share-status.component.html (hunk 1 above)
Comment (ruijadom) [reply Trackovid19-web/src/app/shared/share-status/share-status.component.html]: Yes! i do right now!

**Review by diogoteix — APPROVED** [review]

**Pull request merged**